CONTRIBUTING.md

```diff
@@ -54,5 +54,23 @@ accept your pull requests.
 1.  Lint (and maybe fix) any changes:
 
         npm run fix
-
-[node]: https://nodejs.org/en/
\ No newline at end of file
+        
+## Testing a new feature using CLI
+
+1. After you've written some new code, in order to test it out, you can use the [CLI][CLI]. 
+   
+   For example, the CLI command:
+   
+   `release-please release-pr 
```

Comment:
```suggestion
   `release-please release-pr \
```

Reply:
@ddixit14  The inline code syntax is strange. It has unnecessary backslash.

![image](https://user-images.githubusercontent.com/28604/200019004-880c55dd-c814-49b8-8b2e-d7eeac481eeb.png)

Use multiline code syntax with backslash.